### PR TITLE
VMs: More tweaks, matching IPs, Auth checks

### PR DESCRIPTION
**Pull request by ldemailly**
Verification steps for istio-auth-node-agent

Minor clarification on the main `machineSetup` script

Using host instead of dig as the output is simpler/straightforward

### CODE REVIEW

File: _docs/setup/kubernetes/mesh-expansion.md [hunk 1]
```diff
@@ -201,6 +204,41 @@ or other mesh expansion machines.
...
+```bash
+ps aux |grep istio
+```
+```
+root      6941  0.0  0.2  75392 16820 ?        Ssl  21:32   0:00 /usr/local/istio/bin/node_agent --logtostderr
+root      6955  0.0  0.0  49344  3048 ?        Ss   21:32   0:00 su -s /bin/bash -c INSTANCE_IP=10.150.0.5 POD_NAME=demo-vm-1 POD_NAMESPACE=default exec /usr/local/bin/pilot-agent proxy > /var/log/istio/istio.log istio-proxy
+istio-p+  6999  0.0  0.0  45236  4632 ?        Ss   21:32   0:00 /lib/systemd/systemd --user
+istio-p+  7000  0.0  0.0  61192  1924 ?        S    21:32   0:00 (sd-pam)
+istio-p+  7016  0.0  0.1 215172 12096 ?        Ssl  21:32   0:00 /usr/local/bin/pilot-agent proxy
+istio-p+  7094  4.0  0.3  69540 24800 ?        Sl   21:32   0:37 /usr/local/bin/envoy -c /etc/istio/proxy/envoy-rev1.json --restart-epoch 1 --drain-time-s 2 --parent-shutdown-time-s 3 --service-cluster istio-proxy --service-node sidecar~10.150.0.5~demo-vm-1.default~default.svc.cluster.local
```
Comment (wattli) [inline _docs/setup/kubernetes/mesh-expansion.md]: Can we remove some of irrelevant info here?

Comment (ldemailly) [reply _docs/setup/kubernetes/mesh-expansion.md]: was curious was (sd-pam) was but the rest is relevant no ?

Comment (ldemailly) [reply _docs/setup/kubernetes/mesh-expansion.md]: done

**Comment by ldemailly** [comment]
@wattli in this repo, one needs github approve (well not technically, I can merge it anyway)
https://github.com/istio/istio.github.io/pull/665/files -> review -> approve

**Comment by wattli** [comment]
well, it's interesting that we have different set up for different repos.

**Comment by ldemailly** [comment]
yes I know, it’s one of the many annoyance of multi repos

**Pull request merged**